src/webapp/components/DrivingClusterChip.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
```

Comment:
refactored out from `RequestPopup`

Reply:
heads up although im sure you're aware, we mayyy have merge conflicts- happy to work em out with you!